### PR TITLE
Feat/nrf52

### PR DESCRIPTION
**Pull request by boundlesscalm**

### CODE REVIEW

File: src/features/flows/nRF52833/index.ts [hunk 1]
```diff
@@ -0,0 +1,234 @@
...
+import Info from '../../../common/steps/Info';
+import Learn from '../../../common/steps/Learn';
+import Program from '../../../common/steps/program';
+import { Choice } from '../../../common/steps/program/programSlice';
+import Rename from '../../../common/steps/Rename';
+
+const infoConfig = {
+    title: 'Affordable single-board development kit',
+    markdownContent:
+        '![nRF52833 DK](52833DK.png)  \n&nbsp;  \nThe nRF52833 DK is an affordable single-board development kit for development of Bluetooth® Low Energy, Bluetooth mesh, NFC, Thread and Zigbee applications on the nRF52833 SoC. It also supports development on the nRF52820 SoC.  \n&nbsp;  \n&nbsp;  \n![nRF52833 DK](52833DKTech.png)  \n&nbsp;  \nAn included NFC antenna can be connected to the kit to use the on-chip NFC-A tag functionality. The kit allows access to all I/O and interfaces via connectors and has 4 LEDs and 4 buttons which are user-programmable.  \n&nbsp;  \n[Hardware documentation](https://docs.nordicsemi.com/bundle/ncs-latest/page/zephyr/boards/nordic/nrf52833dk/doc/index.html)',
```
Comment (greg-fer) [inline src/features/flows/nRF52833/index.ts:20]:
```suggestion
        '![nRF52833 DK](52833DK.png)  \n&nbsp;  \nThe nRF52833 DK is an affordable single-board development kit for Bluetooth® Low Energy, Bluetooth Mesh, NFC, Thread, and Zigbee applications on the nRF52833 SoC. It also supports development on the nRF52820 SoC.  \n&nbsp;  \n&nbsp;  \n![nRF52833 DK](52833DKTech.png)  \n&nbsp;  \nAn included NFC antenna can be connected to the kit to use the on-chip NFC-A tag functionality. The kit allows access to all I/O and interfaces via connectors and has four LEDs and four buttons that you can program.  \n&nbsp;  \n[Hardware documentation](https://docs.nordicsemi.com/bundle/ncs-latest/page/zephyr/boards/nordic/nrf52833dk/doc/index.html)',
```

File: src/features/flows/nRF52833/index.ts [hunk 2]
```diff
@@ -0,0 +1,234 @@
...
+};
+
+const programConfig = [
+    {
+        name: 'Hello World',
+        type: 'jlink',
+        description: 'Check that UART is working with this sample.',
+        documentation: {
+            label: 'Hello world',
```
Comment (greg-fer) [inline src/features/flows/nRF52833/index.ts:29]:
```suggestion
            label: 'Hello World',
```

File: src/features/flows/nRF52833/index.ts [hunk 3]
```diff
@@ -0,0 +1,234 @@
...
+            href: 'https://docs.nordicsemi.com/bundle/ncs-2.7.0/page/zephyr/samples/hello_world/README.html',
+        },
+        firmware: [
+            {
+                core: 'Application',
+                file: 'nrf52833dk_hello_world.hex',
+                link: {
+                    label: 'Hello world',
```
Comment (greg-fer) [inline src/features/flows/nRF52833/index.ts:37]:
```suggestion
                    label: 'Hello World',
```

File: src/features/flows/nRF52833/index.ts [hunk 4]
```diff
@@ -0,0 +1,234 @@
...
+                    href: 'https://docs.nordicsemi.com/bundle/ncs-2.7.0/page/zephyr/samples/hello_world/README.html',
+                },
+            },
+        ],
+    },
+    {
+        name: 'Peripheral LED Button Service',
+        type: 'jlink',
+        description:
+            "Check that the LED's and buttons on the DK are working with this sample.",
```
Comment (greg-fer) [inline src/features/flows/nRF52833/index.ts:47]:
```suggestion
            "Check that the LEDs and buttons on the DK are working with this sample.",
```

File: src/features/flows/nRF52833/index.ts [hunk 5]
```diff
@@ -0,0 +1,234 @@
...
+                    label: 'Peripheral LBS',
+                    href: 'https://docs.nordicsemi.com/bundle/ncs-2.7.0/page/nrf/samples/bluetooth/peripheral_lbs/README.html',
+                },
+            },
+        ],
+    },
+    {
+        name: 'Peripheral UART Service',
+        type: 'jlink',
+        description: 'Check that UART over BLE is working.',
```
Comment (greg-fer) [inline src/features/flows/nRF52833/index.ts:66]:
```suggestion
        description: 'Check that UART over Bluetooth® LE is working.',
```

File: src/features/flows/nRF52833/index.ts [hunk 6]
```diff
@@ -0,0 +1,234 @@
...
+        ],
+    },
+] as Choice[];
+
+const verifyConfig = [
+    {
+        ref: 'Hello World',
+        config: {
+            vComIndex: 0,
+            regex: /(\*{3} Booting nRF Connect SDK .* \*{3}\r\nHello World! nrf52833dk_nrf52833)/,
```
Comment (greg-fer) [inline src/features/flows/nRF52833/index.ts:89]:
```suggestion
            regex: /(\*{3} Booting the nRF Connect SDK.* \*{3}\r\nHello World! nrf52833dk_nrf52833)/,
```

File: src/features/flows/nRF52833/index.ts [hunk 7]
```diff
@@ -0,0 +1,234 @@
...
+        },
+    },
+    {
+        ref: 'Peripheral LED Button Service',
+        config: {
+            vComIndex: 0,
+            regex: /(\*{3} Booting nRF Connect SDK .* \*{3}\r\nStarting Bluetooth Peripheral LBS example)/,
```
Comment (greg-fer) [inline src/features/flows/nRF52833/index.ts:96]:
```suggestion
            regex: /(\*{3} Booting the nRF Connect SDK.* \*{3}\r\nStarting the Bluetooth Peripheral LBS sample)/,
```

File: src/features/flows/nRF52833/index.ts [hunk 8]
```diff
@@ -0,0 +1,234 @@
...
+        },
+    },
+    {
+        ref: 'Peripheral UART Service',
+        config: {
+            vComIndex: 0,
+            regex: /(\*{3} Booting nRF Connect SDK .* \*{3}\r\nStarting Nordic UART service example)/,
```
Comment (greg-fer) [inline src/features/flows/nRF52833/index.ts:103]:
```suggestion
            regex: /(\*{3} Booting the nRF Connect SDK.* \*{3}\r\nStarting the Nordic UART service sample)/,
```

File: src/features/flows/nRF52833/index.ts [hunk 9]
```diff
@@ -0,0 +1,234 @@
...
+            },
+        ],
+    },
+];
+
+const learnConfig = [
+    {
+        label: 'Developer Academy',
+        description:
+            'Speed up your wireless IoT learning journey with Nordic devices.',
```
Comment (greg-fer) [inline src/features/flows/nRF52833/index.ts:178]: The original line is fine for nRF91 LTE devices, but here we're targeting a broader spectrum of protocols, no?
```suggestion
            'Get the know-how to build wireless products using Nordic Semiconductor solutions.',
```

File: src/features/flows/nRF52/index.ts [hunk 10]
```diff
@@ -0,0 +1,234 @@
...
+            },
+        ],
+    },
+];
+
+const learnConfig = [
+    {
+        label: 'Developer Academy',
+        description:
+            'Speed up your wireless IoT learning journey with Nordic devices.',
```
Comment (greg-fer) [inline src/features/flows/nRF52/index.ts:178]: The original line is fine for nRF91 LTE devices, but here we're targeting a broader spectrum of protocols, no?
```suggestion
            'Get the know-how to build wireless products using Nordic Semiconductor solutions.',
```

File: src/features/flows/nRF52833/index.ts [hunk 11]
```diff
@@ -0,0 +1,234 @@
...
+            'Learn about the application development in the nRF Connect SDK and Zephyr.',
+        link: {
+            label: 'Application development',
+            href: 'https://docs.nordicsemi.com/bundle/ncs-latest/page/nrf/app_dev.html',
+        },
+    },
+    {
+        label: 'Developing with nRF52 Series',
+        description:
+            'Device-specific information about features, DFU solution, development and much more.',
```
Comment (greg-fer) [inline src/features/flows/nRF52833/index.ts:196]:
```suggestion
            'Device-specific information about features, DFU solution, and development.',
```

File: src/features/flows/nRF52840/index.ts [hunk 12]
```diff
@@ -0,0 +1,234 @@
...
+        '![nRF52840 DK](52840DK.png)  \n&nbsp;  \nThe nRF52840 DK is a versatile single-board development kit for Bluetooth Low Energy, Bluetooth mesh, Thread, Zigbee, 802.15.4, ANT and 2.4 GHz proprietary applications on the nRF52840 SoC. It is the recommended Nordic development kit for Amazon Sidewalk. It also supports development on the nRF52811 SoC.  \n&nbsp;  \n&nbsp;  \n![Technologies](52840DKTech.png)  \n&nbsp;  \nThe nRF52840 DK can also be used for Matter over Thread where Thread is used for transport and Bluetooth LE for commissioning. Matter devices based on Thread are required to feature Bluetooth LE concurrently to enable adding new devices to a network.  \n&nbsp;  \n[Hardware documentation](https://docs.nordicsemi.com/bundle/ug_nrf52840_dk/page/UG/dk/intro.html)',
+};
+
+const programConfig = [
+    {
+        name: 'Hello World',
+        type: 'jlink',
+        description: 'Check that UART is working with this sample.',
+        documentation: {
+            label: 'Hello world',
```
Comment (greg-fer) [inline src/features/flows/nRF52840/index.ts:29]:
```suggestion
            label: 'Hello World',
```

File: src/features/flows/nRF52840/index.ts [hunk 13]
```diff
@@ -0,0 +1,234 @@
...
+            href: 'https://docs.nordicsemi.com/bundle/ncs-2.7.0/page/zephyr/samples/hello_world/README.html',
+        },
+        firmware: [
+            {
+                core: 'Application',
+                file: 'nrf52840dk_hello_world.hex',
+                link: {
+                    label: 'Hello world',
```
Comment (greg-fer) [inline src/features/flows/nRF52840/index.ts:37]:
```suggestion
                    label: 'Hello World',
```

File: src/features/flows/nRF52840/index.ts [hunk 14]
```diff
@@ -0,0 +1,234 @@
...
+import Info from '../../../common/steps/Info';
+import Learn from '../../../common/steps/Learn';
+import Program from '../../../common/steps/program';
+import { Choice } from '../../../common/steps/program/programSlice';
+import Rename from '../../../common/steps/Rename';
+
+const infoConfig = {
+    title: 'Versatile single-board development kit',
+    markdownContent:
+        '![nRF52840 DK](52840DK.png)  \n&nbsp;  \nThe nRF52840 DK is a versatile single-board development kit for Bluetooth Low Energy, Bluetooth mesh, Thread, Zigbee, 802.15.4, ANT and 2.4 GHz proprietary applications on the nRF52840 SoC. It is the recommended Nordic development kit for Amazon Sidewalk. It also supports development on the nRF52811 SoC.  \n&nbsp;  \n&nbsp;  \n![Technologies](52840DKTech.png)  \n&nbsp;  \nThe nRF52840 DK can also be used for Matter over Thread where Thread is used for transport and Bluetooth LE for commissioning. Matter devices based on Thread are required to feature Bluetooth LE concurrently to enable adding new devices to a network.  \n&nbsp;  \n[Hardware documentation](https://docs.nordicsemi.com/bundle/ug_nrf52840_dk/page/UG/dk/intro.html)',
```
Comment (greg-fer) [inline src/features/flows/nRF52840/index.ts:20]: Matter is a separate protocol and needs to be mentioned in the list, even if underneath it uses Matter over Thread.
```suggestion
        '![nRF52840 DK](52840DK.png)  \n&nbsp;  \nThe nRF52840 DK is a versatile single-board development kit for Bluetooth® Low Energy, Bluetooth Mesh, Matter, Thread, Zigbee, 802.15.4, ANT, and 2.4-GHz proprietary applications on the nRF52840 SoC. It is the recommended Nordic development kit for Amazon Sidewalk. It also supports development on the nRF52811 SoC.  \n&nbsp;  \n&nbsp;  \n![Technologies](52840DKTech.png)  \n&nbsp;  \nThe nRF52840 DK can be used for Matter over Thread where Thread is used for transport and Bluetooth LE for commissioning. Matter devices based on Thread are required to feature Bluetooth LE concurrently to enable adding new devices to a network.  \n&nbsp;  \n[Hardware documentation](https://docs.nordicsemi.com/bundle/ug_nrf52840_dk/page/UG/dk/intro.html)',
```

File: src/features/flows/nRF52840/index.ts [hunk 15]
```diff
@@ -0,0 +1,234 @@
...
+                    href: 'https://docs.nordicsemi.com/bundle/ncs-2.7.0/page/zephyr/samples/hello_world/README.html',
+                },
+            },
+        ],
+    },
+    {
+        name: 'Peripheral LED Button Service',
+        type: 'jlink',
+        description:
+            "Check that the LED's and buttons on the DK are working with this sample.",
```
Comment (greg-fer) [inline src/features/flows/nRF52840/index.ts:47]:
```suggestion
            "Check that the LEDs and buttons on the DK are working with this sample.",
```

File: src/features/flows/nRF52840/index.ts [hunk 16]
```diff
@@ -0,0 +1,234 @@
...
+                    label: 'Peripheral LBS',
+                    href: 'https://docs.nordicsemi.com/bundle/ncs-2.7.0/page/nrf/samples/bluetooth/peripheral_lbs/README.html',
+                },
+            },
+        ],
+    },
+    {
+        name: 'Peripheral UART Service',
+        type: 'jlink',
+        description: 'Check that UART over BLE is working.',
```
Comment (greg-fer) [inline src/features/flows/nRF52840/index.ts:66]:
```suggestion
        description: 'Check that UART over Bluetooth® LE is working.',
```

File: src/features/flows/nRF52840/index.ts [hunk 17]
```diff
@@ -0,0 +1,234 @@
...
+        ],
+    },
+] as Choice[];
+
+const verifyConfig = [
+    {
+        ref: 'Hello World',
+        config: {
+            vComIndex: 0,
+            regex: /(\*{3} Booting nRF Connect SDK .* \*{3}\r\n\*{3} Using Zephyr OS .* \*{3}\r\nHello World! nrf52840dk\/nrf52840)/,
```
Comment (greg-fer) [inline src/features/flows/nRF52840/index.ts:89]:
```suggestion
            regex: /(\*{3} Booting the nRF Connect SDK.* \*{3}\r\n\*{3} Using the Zephyr OS.* \*{3}\r\nHello World! nrf52840dk\/nrf52840)/,
```

**Review by greg-fer — CHANGES_REQUESTED** [review]

File: src/features/flows/nRF52/index.ts [hunk 18]
```diff
@@ -0,0 +1,234 @@
...
+        config: {
+            vComIndex: 0,
+            regex: /(\*{3} Booting nRF Connect SDK .* \*{3}\r\n\*{3} Using Zephyr OS .* \*{3}\r\nHello World! nrf52dk\/nrf52832)/,
+        },
+    },
+    {
+        ref: 'Peripheral LED Button Service',
+        config: {
+            vComIndex: 0,
+            regex: /(\*{3} Using nRF Connect SDK .* \*{3}\r\n\*{3} Using Zephyr OS .* \*{3}\r\nStarting Bluetooth Peripheral LBS example)/,
```
[the remark this answers is not in the document]
Comment (boundlesscalm) [reply src/features/flows/nRF52/index.ts:96]: Cannot fix this. This is defined by the sample and has to be an exact match.

File: src/features/flows/nRF52/index.ts [hunk 19]
```diff
@@ -0,0 +1,234 @@
...
+        ],
+    },
+] as Choice[];
+
+const verifyConfig = [
+    {
+        ref: 'Hello World',
+        config: {
+            vComIndex: 0,
+            regex: /(\*{3} Booting nRF Connect SDK .* \*{3}\r\n\*{3} Using Zephyr OS .* \*{3}\r\nHello World! nrf52dk\/nrf52832)/,
```
[the remark this answers is not in the document]
Comment (boundlesscalm) [reply src/features/flows/nRF52/index.ts:89]: Same here with me not being able to change this

File: src/features/flows/nRF52/index.ts [hunk 20]
```diff
@@ -0,0 +1,234 @@
...
+        config: {
+            vComIndex: 0,
+            regex: /(\*{3} Using nRF Connect SDK .* \*{3}\r\n\*{3} Using Zephyr OS .* \*{3}\r\nStarting Bluetooth Peripheral LBS example)/,
+        },
+    },
+    {
+        ref: 'Peripheral UART Service',
+        config: {
+            vComIndex: 0,
+            regex: /(\*{3} Using nRF Connect SDK .* \*{3}\r\n\*{3} Using Zephyr OS .* \*{3}\r\nStarting Nordic UART service example)/,
```
[the remark this answers is not in the document]
Comment (boundlesscalm) [reply src/features/flows/nRF52/index.ts:103]: Same here with me not being able to change this

**Pull request merged**